annotation/functions.scm

```diff
@@ -885,3 +885,16 @@
            (equal? (cog-type (gar memblink)) 'GeneNode)))
     (cog-incoming-by-type go-term 'MemberLink))
 )
+
+(define (find-go-protein go-term)
+"
+  find-go-protein GO-TERM
+
+  Find the proteins associate with GO-TERM via a MemberLink.
+"
+  (filter
```

Comment:
I think using the following pattern matcher query would be more efficient

```
(Get 
    (TypedVariable (Variable "$prot") (TypedNode "MoleculeNode"))
    (Member 
       (Variable "$prot")
       go-term)
) 
```

Reply:
Thanks, I've just changed it